setup.py

```diff
@@ -5,5 +5,5 @@
     name='pylibui',
     version='0.0.1',
     description='Python wrapper for libui',
-    packages=['pylibui', 'pylibui.libui']
+    packages=['pylibui', 'pylibui.libui', 'pylibui.controls']
```

Comment:
Please revert this one. This PR is not for adding pylibui.controls to setup.py. If you think this is necessary, please open an issue about that and we can create a new PR for this.